packages/cohttp-async/cohttp-async.2.5.7/opam

```diff
@@ -0,0 +1,60 @@
+opam-version: "2.0"
+maintainer: "anil@recoil.org"
+authors: [
+  "Anil Madhavapeddy"
+  "Stefano Zacchiroli"
+  "David Sheets"
+  "Thomas Gazagnaire"
+  "David Scott"
+  "Rudi Grinberg"
+  "Andy Ray"
+]
+synopsis: "CoHTTP implementation for the Async concurrency library"
+description: """
+An implementation of an HTTP client and server using the Async
+concurrency library. See the `Cohttp_async` module for information
+on how to use this.  The package also installs `cohttp-curl-async`
+and a `cohttp-server-async` binaries for quick uses of a HTTP(S)
+client and server respectively.
+"""
+license: "ISC"
+tags: ["org:mirage" "org:xapi-project"]
+homepage: "https://github.com/mirage/ocaml-cohttp"
+doc: "https://mirage.github.io/ocaml-cohttp/"
+bug-reports: "https://github.com/mirage/ocaml-cohttp/issues"
+depends: [
+  "ocaml" {>= "4.04.1"}
+  "dune" {>= "1.1.0"}
+  "async_kernel" {>= "v0.14.0"}
+  "async_unix" {>= "v0.14.0"}
+  "async" {>= "v0.14.0"}
+  "base" {>= "v0.11.0"}
+  "core" {with-test}
```

Comment:
```suggestion
  "core" {with-test & < "v0.15.0"}
```
Due to
```
# File "examples/async/hello_world.ml", line 39, characters 5-16:
# 39 |   |> Command.run
#           ^^^^^^^^^^^
# Alert deprecated: Command.run
# [since 2021-03] Use [Command_unix]
# File "examples/async/hello_world.ml", line 39, characters 5-16:
# 39 |   |> Command.run
#           ^^^^^^^^^^^
# Error: This expression has type [ `Use_Command_unix ]
#        but an expression was expected of type Command.t -> 'a
```